oaknational/Oak-Web-Application
# PUPIL-334: Serve quiz images directly from cloudinary

## Description

* Points oak-components at its new home under the oaknational npm scope
* Updates images under the new quiz UI to use `OakCloudinaryImage` which serves up resized/responsive images directly from Cloudinary

https://www.notion.so/oaknationalacademy/Configure-OWA-for-new-OakImage-arrangement-d32790e622db476385963421a867b4b7?pvs=4

## How to test

1. Go to https://deploy-preview-2211--oak-web-application.netlify.thenational.academy/pupils/programmes/maths-secondary-ks3/units/graphical-representations-of-data/lessons/constructing-bar-charts-by-utilising-technology
2. Take the quizzes
3. You should see that the images are snappy to load
